container/kubernetes/onboarding/README.md

```diff
@@ -88,16 +88,23 @@ CreateDeployments:
        	GitTag: <MACHINE_CONFIG_TAG_OR_COMMITID_OR_BRANCH>
 ```
 6. Initiate the Deployment process based on three actions; create, update or delete.
+   AND gatewaydeployment yaml file path.
 ```
-python3 deviceonboarding.py --action create 
+python3 deviceonboarding.py --action create --deployfilepath gatewaydeployment.yaml
+OR
+python3 deviceonboarding.py -a create -d gatewaydeployment.yaml
+===================================================================================
+python3 deviceonboarding.py --action update --deployfilepath gatewaydeployment.yaml
 OR 
-python3 deviceonboarding.py --action update
-OR 
-python3 deviceonboarding.py --action delete
+python3 deviceonboarding.py -a update -d gatewaydeployment.yaml
+===================================================================================
+python3 deviceonboarding.py --action delete --deployfilepath gatewaydeployment.yaml
+OR
+python3 deviceonboarding.py -a delete -d gatewaydeployment.yaml
 ```
 7. Once the onboarding script it initiated it will ask you for OISP platform credentials.
 ```
-Gateway:~/tools $python3 deviceonboarding.py --action delete
+Gateway:~/tools $python3 deviceonboarding.py -a delete -d gatewaydeployment.yaml
```

Comment:
Same as comment above, I would prefer to use "--action" and "--deployfilepath" flags in documentation unless there is a good reason not to.

Reply:
will take care, Just when we use it tool more as we have to type "--action" than "-a" same for "--deployfilepath" than "-d"

Reply:
Agree, the short-hand is likely to get used more but users can work this bit out for themselves and the long-hand alternative makes more sense for documentation IMO.